apis/python/src/tiledbsoma/io/spatial/outgest.py

```diff
@@ -135,9 +139,10 @@ def to_spatial_data_points(
 def to_spatial_data_shapes(
     points: PointCloudDataFrame,
     *,
+    key: str,
```

Comment:
Can you add docstrings for this one. Also could you explain it's use?

Reply:
SpatialData requires that you always define a transformation on an object, but we don't. I hit this issue when attempting to transform an entire Scene. This change makes it so I can set a transformation from the object to itself. I just needed to add in a parameter that specifies the name of the object inside SpatialData.

For additional context, these are helper functions. I plan to move them outside of `outgest.py` to a spatial data util module in a future PR. They should only be accessed inside functions that export a full `SpatialData` class.